src/resources/postcss/views/day/_event.pcss

```diff
@@ -0,0 +1,86 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	 * Event
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-calendar-day__event {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: var(--spacer-5);
```

Comment:
all of the spacers in this selector have changed, check out https://github.com/moderntribe/the-events-calendar/pull/2583/files#diff-ff3de376ecb4382618c6419ad7a5b4f4R8 to see what i've done to match designs.